src/server/pps/server/worker_rc.go

```diff
@@ -100,6 +100,12 @@ func (a *apiServer) workerPodSpec(options *workerOptions) (v1.PodSpec, error) {
 		Name:  "PACHD_POD_NAMESPACE",
 		Value: a.namespace,
 	}}
+	if a.env.NewStorageLayer {
```

Comment:
It seems like it would be better to always propagate the feature flag environment variables, not just the ones that are true. Although I don't see a future where a feature flag is enabled by default in the service environment config, I think it would be a good precedent to always propagate global environment variables so we will have consistent environments across the containers.

Reply:
Actually, I think this may be fine, and preferable, if the feature flag environment variable is only set when it is true in the pachd deployment. Is the new feature flag only being exposed by manually setting it in the deploy manifest?

Reply:
Yeah, agree with second comment, as long as feature flags are always default false (which I think they should be) then the only value people will explicitly set is `true` so I think this makes sense. The feature flag is currently only able to be set manually, I could patch through a flag for `pachctl deploy` though if you think that's worth it.

Reply:
Okay, this should probably be a deployment parameter eventually, but I don't think it is really important now. I think it would make sense to hold off on that until we have templated deployments.